uimaj-core/src/main/java/org/apache/uima/cas/impl/CASImpl.java

```diff
@@ -1117,6 +1117,10 @@ public boolean isCasLocked() {
    */
   private volatile FSIterator<Annotation> docAnnotIter = null;
   
+  // UIMA-6199 provides access to non-indexed doc annot 
+  //   to allow sofa setting to set the "length"
+  private volatile Annotation deserialized_doc_annot_not_indexed = null;
```

Comment:
Length of what?

Reply:
I'll add a comment.  This is the length of the sofa data when it's a string.  (Have to look to see if it's set at all for the other forms of sofa data...)